src/base/static/components/place-detail/place-detail-editor.js

```diff
@@ -62,7 +62,18 @@ class PlaceDetailEditor extends Component {
         fields = fields.set(
           field.name,
           Map()
-            .set("value", this.props.place[field.name])
+            .set(
+              "value",
+              field.name === "private"
+                ? // Private fields will be returned with a true or false value,
+                  // which should be mapped to the "yes"/"no" values of the field
+                  // which manipulates the private setting.
+                  // TODO: This should be better encapsulated.
+                  this.props.place["private"]
```

Comment:
I know this is not great-- at some point I want to refactor to create an actual field type for manipulating the `private` flag, where all this logic is encapsulated.